flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  inputs = {
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    pre-commit-hooks.url = "github:cachix/pre-commit-hooks.nix";
```

Comment:
In case you haven't seen it, note that we already have existing `pre-commit` hooks in https://github.com/polarity-lang/polarity/blob/main/scripts/git-hooks/pre-commit.

Reply:
Oh I haven’t seen that. Then I will remove all that jazz.

Reply:
Or integrate/change the current setup if desirable :)